sdk/include/opentelemetry/sdk/trace/tracer.h

```diff
@@ -60,8 +60,8 @@ class Tracer final : public trace_api::Tracer, public std::enable_shared_from_th
   Sampler &GetSampler() { return context_->GetSampler(); }
 
 private:
-  std::shared_ptr<sdk::trace::TracerContext> context_;
   std::shared_ptr<InstrumentationLibrary> instrumentation_library_;
```

Comment:
Could we document the order requirement in the comment as well?

Reply:
good point, documented it now.